code/previous_artists/arlo.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="eng"></html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Project 2: Noemi's Music Magazine</title>
+    <link rel="stylesheet" href="../style.css"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Abril+Fatface&family=Amiri&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+    <header> 
+      <h1>Noemi's Music Magazine <img src="../images/logo.jpg" alt="Logo" height="40px"> </h1>
+      <nav>
+        <div><a href="../index.html">Home</a></div>
+        <div>Concerts</div>
+        <div>Events</div>
+        <div>About</div>
+      </nav>
+    </header>
+
+<img class="img-artistcards" src ="../images/arlo.jpg" alt="Arlo Parks">
+<h4>Arlo Parks</h4>
+<p class="author">by Noemi, 01.01.2021</p>
+    <p class="text-artist">Arlo Parks is a 20 year old poet/singer from London. In her words she spent most of school feeling like that black kid who couldn't 
+    dance for shit, listening to too much emo music and crushing on some girl in her Spanish class. Her songs are confessional and tender, 
+    mainly inspired by Portishead and Earl Sweatshirt. </p>
```

Comment:
Indentation ;) Siblings should have the same level of indentation. Goes for all HTML files